projects/racket/project.yaml

```diff
@@ -0,0 +1,13 @@
+homepage: "https://racket-lang.org"
+primary_contact: "pocmatos@gmail.com"
+auto_ccs:
+  - samth@racket-lang.org
+sanitizers:
+ - undefined
+ - memory:
+    experimental: True
+ - address:
+    experimental: True
```

Comment:
ASan is a core sanitizer, we don't recommend using an experimental flag here.
Also, we no longer recommend experimental for anything (ideally should just run the fuzzer locally with that config for a few secs), can you remove it from the two sanitizers.

Reply:
Done - thanks for the comments.